Config/Inc/config.h

```diff
@@ -38,16 +38,17 @@ typedef enum SafetyStatus_e {SAFE = 0, DANGER = 1, OVERVOLTAGE = 2, UNDERVOLTAGE
 // MAX and MIN limits of lithium ion cells
 #define MIN_VOLTAGE_CHARGING_LIMIT      2550        // Under voltage limit for charging (milliVolts) 
 #define MIN_VOLTAGE_LIMIT				2700		// Under voltage limit (milliVolts)	(actual min: 2.5V)
-#define MAX_VOLTAGE_LIMIT				4000		// Over voltage limit (milliVolts)		(actual max: 4.2V)
+#define MAX_VOLTAGE_LIMIT				4100		// Over voltage limit (milliVolts)		(actual max: 4.2V)
+#define CHARGE_DISABLE_VOLTAGE          4000        // Voltage to stop charging at
```

Comment:
Could also include something like
```C
#if MAX_VOLTAGE_LIMIT - 100 < CHARGE_DISABLE_VOLTAGE
#error "Charging maximum voltage is too close to voltage trip limit!"
#endif
```
Or do something with the `#pragma` you found